### PR TITLE
Added game whitelist

### LINKED ISSUE #7: [feature] Add game whitelist for live role
Add a whitelist of games for what can get the live role

Could be a csv list of gameids in a key in ;config

### PR DESCRIPTION
**Pull request by notfooddev**
Add whitelist of games for which a stream live notification is given. If none are set, there is no whitelist applied.

This change was tested.

### CODE REVIEW

File: src/interfaces/serverSettings.ts [hunk 1]
```diff
@@ -14,5 +14,5 @@ export default interface ServerSettings {
 	streamShout: string|null;
 
 	adminRole: string|null;
-	moderatorRole: string|null;
+	moderatorRole: string | null;
```
Comment (WesVleuten) [inline src/interfaces/serverSettings.ts]: Weird edit

File: src/lib/twitch.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import config from './config';
```
Comment (WesVleuten) [inline src/lib/twitch.ts:1]: Invalid indentation

File: src/repository/whiteListedGames.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+export default class WhiteListedGamesRepository {
+
+	static async GetByGuildId(guildId: string | undefined): Promise<whiteListedGame[]|null> {
+		if (!guildId) {
+			return null;
+		}
+		const database = Database.getInstance()
+
+    return await database.query<whiteListedGame[]>("SELECT * FROM WhiteListedGames WHERE guildId = ?", [guildId]);
```
Comment (WesVleuten) [inline src/repository/whiteListedGames.ts]: Indentation

**Review by WesVleuten — CHANGES_REQUESTED** [review]
Looks great, just a couple small fixes needed

File: src/lib/twitch.ts (hunk 2 above)
Comment (notfooddev) [reply src/lib/twitch.ts:1]: Don't see what's wrong here, import is copied from database.ts in the same dir

Comment (WesVleuten) [reply src/lib/twitch.ts:1]: It was more a comment on the whole file, it uses 2 space indentation instead of tabs

**Review by WesVleuten — APPROVED** [review]

**Pull request merged**